morpheushelper/cogs/mod.py

```diff
@@ -419,9 +420,9 @@ async def kick(self, ctx: Context, member: Member, *, reason: str):
 
         if not ctx.guild.me.guild_permissions.kick_members:
             raise CommandError(translations.cannot_kick_permissions)
-
-        if member.top_role >= ctx.guild.me.top_role or member.id == ctx.guild.owner_id:
-            raise CommandError(translations.cannot_kick)
+        if member_on_server:
```

Comment:
let's add a line here and rearrange the if statement.
```python
member_on_server: bool = isinstance(user, Member)
if member_on_server and (member.top_role >= ctx.guild.me.top_role or member.id == ctx.guild.owner_id):
    pass
```

Reply:
@Tert0 is there a reason for not rearranging the if statement.

Reply:
Did you mean that?
```python
member_on_server: bool = isinstance(member, Member)
        if member_on_server and member.top_role >= ctx.guild.me.top_role or member.id == ctx.guild.owner_id:
            raise CommandError(translations.cannot_kick)
```

Reply:
> ```python
> member_on_server: bool = isinstance(user, Member)
> if member_on_server and (member.top_role >= ctx.guild.me.top_role or member.id == ctx.guild.owner_id):
>     pass
> ```
@Paul-Brauckmann
Should not be thrown an error(`CommandError(translations.cannot_kick)`)?

Reply:
@Tert0 I'm sry for your inconvenience to try to understand what i want. Yes i mean u should write this:
```python
if member_on_server and (member.top_role >= ctx.guild.me.top_role or member.id == ctx.guild.owner_id):
    raise CommandError(translations.cannot_kick)
```
The thing is that i only wanted to highlight the if statement and not the other parts but also give you a valide python syntax.... Now I see, that I should clarify this the next time.